clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -939,6 +943,15 @@ DPCT_ENUM_OPTION(
     llvm::cl::CommaSeparated, llvm::cl::value_desc("value"),
     llvm::cl::cat(DPCTCat), llvm::cl::cat(DPCTCodeGenCat), llvm::cl::ZeroOrMore)
 
+DPCT_FLAG_OPTION(
+    UseCompat, clang::dpct::DpctOptionClass::OC_Attribute,
+    DPCT_OPTION_ACTIONS(clang::dpct::DpctActionKind::DAK_Analysis,
+                        clang::dpct::DpctActionKind::DAK_Migration),
+    "use-compat",
+    llvm::cl::desc("Call SYCLcompat function in migrated code "
```

Comment:
--use-syclcompat
Use SYCLcompat header-only library (syclcompat:: namespace) instead of DPCT helper functions (dpct:: namespace) to assist the migration of input source code

Reply:
1). Let's add new option:   --use-syclcompat,  
2) it should not be used together with: --gen-helper-function, --helper-function-dir, --helper-function-preference
3) it should be listed in -help-info
4) it should